x-pack/filebeat/module/salesforce/module.yml

```diff
@@ -0,0 +1,3 @@
+dashboards:
+- id: Filebeat-salesforce-login-Dashboard
+  file: Filebeat-salesforce-login.json

```

Comment:
This file is missing. I suggest to remove this entry until you create a dashboard presenting collected Salesforce data.

Reply:
Right, will do